src/styles/blocks/calendar-block.scss

```
@@ -0,0 +1,46 @@
+.calendar {
+  padding: $calendar-padding;
+  gap: $gap-size;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+
+  @each $day, $mod in $week-days {
+    &--start-day--#{$day} :first-child {
+      margin-left: (($gap-size * $mod) + ($cell-size * $mod));
+    }
+  }
+
+  @for $last-day from 28 through 31 {
+    &--month-length--#{$last-day} :nth-child(#{$last-day + 1} + n) {
+      opacity: 0;
```

no needs to display redundant cells
```suggestion
      display: none;
```